openedx/edx-ora2
# fix: remove blank template, filter out peer template so it doesn't display twice

**TL;DR -** Remove blank template and make peer assessment the default ORA template 

JIRA: [AU-75](https://2u-internal.atlassian.net/browse/AU-75)

**What changed?**

- Remove blank ORA template
- implement optional template function to filter out "duplicate" peer assessment (the list of templates includes peer-assessment, and we are inserting "peer-assessment" as the "default" template)

**Testing Instructions**

Check out this branch and https://github.com/openedx/edx-platform/pull/30705. Install local ORA.

FYI: @openedx/content-aurora
